### PR TITLE
Python SDK Account Module + Tests (T1702)

### PR DESCRIPTION
**Pull request by vchawla3**

### CODE REVIEW

**Comment by kkushagra** [comment]
Move the accounts tests to a tests/account folder. Also create an empty __init__.py in the test/account folder. This will let you run the tests locally and the build environment will also accept it.

**Comment by vchawla3** [comment]
Hi, I have moved the files and added an __init__.py.

File: tests/accounts/test_share.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
+#	THESE METHODS IN SHARE.PY ARE ALL TESTED THROUGH TEST_BUCKETS.PY
```
Comment (kkushagra) [inline tests/accounts/test_share.py]: The whole test_share.py is commented. If it is not required then delete it, else test it and commit it again.

File: tests/accounts/test_devices.py [hunk 2]
```diff
@@ -0,0 +1,157 @@
+from tago import Tago
+from tago.account.devices import Devices  as Devices
+import os
+
+TOKEN = os.environ.get('TAGO_TOKEN_ACCOUNT') or 'a0030850-d585-4063-be6c-f59fdd7046c8'
+DEBUG_MESSAGE = 'The response to {} \n{}'
+
+def test_device_list():
+    # print dir(Devices)
```
Comment (kkushagra) [inline tests/accounts/test_devices.py]: Dangling comment, if it is not useful then remove it.

File: tests/accounts/test_dashboards_widgets.py [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+    print DEBUG_MESSAGE.format('dashboards.widgets deletion', dashboardsWidgetsResult)
+
+    if dashboardsWidgetsResult['status']:
+        assert True
+    else:
+        assert False
+
+    dashboard.delete(test_id)
+
+#test_dashboards_widgets_sendData()
```
Comment (kkushagra) [inline tests/accounts/test_dashboards_widgets.py]: Dangling comment, if it is not useful then remove it.

File: tago/account/__init__.py [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+from devices import Devices
+from buckets import Buckets
+
+API_TAGO = os.environ.get('TAGO_SERVER') or 'https://api.tago.io'
+REALTIME = os.environ.get('TAGO_REALTIME') or 'https://realtime.tago.io'
+
+class Account:
+    def __init__(self, token):
+        self.token = token
+        # Not sure about this next line..the 'default headers part'
```
Comment (kkushagra) [inline tago/account/__init__.py]: The default_header is correct and can be used as specified. This comment is not needed, kindly remove it.

**Comment by vitorfdl** [comment]
you need to solve the conflicts, then I can merge it

**Pull request merged**